dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.auditWebProbes1.0.javaee.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName =io.openliberty.auditWebProbes1.0.javaee
+visibility = private
+IBM-Install-Policy: when-satisfied
+
+IBM-Provision-Capability:  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.audit1.0.javaee))"
```

Comment:
This filter doesn't make any sense.  Just move the bundles from this auto feature to the audit1.0.javaee auto feature.